java-common-protos/.repo-metadata.json

```diff
@@ -1,14 +1,14 @@
 {
   "api_shortname": "common-protos",
   "name_pretty": "Common Protos",
-  "product_documentation": "n/a",
-  "api_description": "n/a",
+  "product_documentation": "https://github.com/googleapis/sdk-platform-java/tree/main/java-common-protos",
```

Comment:
This might be best, but have we considered https://github.com/googleapis/api-common-protos since it's language-agnostic?

Reply:
Makes sense! updated.